zstacklib/zstacklib/utils/sanlock.py

```
@@ -1,20 +1,60 @@
+        r, _, e = repair_delta_lease_if_corrupted(vg_name, host_id)
+    else:
+        r, _, e = direct_init_resource2(vg_name, resource_name, offset)
+
+    if r != 0:
+        return "failed to init sanlock lease, err %s" % e
+
+
+def request_repair_metadata(vg_name, specify_host_by_id=False, **lease_struct):
+    url = report.Report.url
+    host_uuid = report.Report.serverUuid
+    struct = {"primaryStorageUuid": vg_name, "hostUuid": host_uuid, "leaseStruct": lease_struct, "specifyHostById": specify_host_by_id}
```

Pylint (3.3.7)

[convention] Line too long (135/100)

(C0301)